### PR TITLE
fix #78865 vsemayki.ru

### LINKED ISSUE #78865: Broken vk mini app vsemayki.ru
[//]: # (***You can delete or ignore strings starting with "[//]:" They will not be visible either way.)

***Description***:
* **Current behaviour**: broken application https://vk.com/services?w=app6800340_6049413

[//]: # application based on vk mini apps - https://vk.com/dev/vk_apps_docs, and auto add to url search params **vk_ref** on launch application

[//]: # ScreenShot
<details><summary>Screenshot:</summary>

image

</details><br/>

* **Expected behaviour**: 

[//]: # (Substitute this line with a description of what should happen normally. If needed, provide a screenshot below, same as above)

<details><summary>Screenshot:</summary>

image

</details><br/>

***Steps to reproduce the problem***:

[//]: # (Substitute this line with a step-by-step instruction how to reproduce the issue)

- Go to https://vk.com/services?act=search&q=vsemayki
- click application Vsemayki

***System configuration***

**Filters:**

[//]: # (Substitute this line with the list of your active filters, separated by commas)

[//]: # (Please enter the correct values for your case to the table below)

Information                            | Value
---                                    | ---
Operating system:                      | Ubuntu
Operating system version (Android/iOS) | ?
Browser:                               | Chrome v81
AdGuard version:                       | Версия 3.5.31
Filters enabled:                       | ?
AdGuard mode (Android only):           | VPN / Proxy (manual/auto)
Filtering quality (Android only):      | High-quality / High-speed / Simplified
HTTPS filtering (Android only):        | On (Default/Blacklist mode) / Off
Simplified filters (iOS only)          | On / Off
AdGuard DNS:                           | None / Default / Family Protection
Stealth mode options (Windows only)    | ?
Helpdesk ID (if exists):               | ?

### PR DESCRIPTION
**Pull request by akeinhell**
https://github.com/AdguardTeam/AdguardFilters/issues/78865

### CODE REVIEW

File: RussianFilter/sections/adservers.txt [hunk 1]
```diff
@@ -4412,7 +4412,7 @@ us6odtasve.ru
 ||vsekiski.org^$third-party
 ||vsem.ru/img$third-party
 ||vsemayki.ru*/?ref=$third-party
-||vsemayki.ru/*ref=$third-party
```
Comment (zloyden) [inline RussianFilter/sections/adservers.txt:4415]: @Alex-302 do we need these rules?

Comment (akeinhell) [reply RussianFilter/sections/adservers.txt:4415]: @zloyden Not sure. 
can I delete these rules?

Comment (zloyden) [reply RussianFilter/sections/adservers.txt:4415]: Well, you can temporary add `||vsemayki.ru/*ref=$third-party,badfilter` to the User rules.
But these rules are really old and @Alex-302 knows more about them. Let's wait for his decision.

**Review by Alex-302 — APPROVED** [review]

File: RussianFilter/sections/adservers.txt (hunk 1 above)
Comment (Alex-302) [reply RussianFilter/sections/adservers.txt:4415]: The rulses are old, but they was added to block ads from `vsemayki.ru`

Comment (zloyden) [reply RussianFilter/sections/adservers.txt:4415]: Ok.

Comment (zloyden) [reply RussianFilter/sections/adservers.txt:4415]: @akeinhell update should be available in 1-2 hours.

Comment (akeinhell) [reply RussianFilter/sections/adservers.txt:4415]: Thanks

**Pull request merged**